R-package/README.md

```diff
@@ -454,13 +454,17 @@ cd R-package/tests
 RDvalgrind \
     --no-readline \
     --vanilla \
-    -d valgrind \
+    -d "valgrind --tool=memcheck --leak-check=full --track-origins=yes" \
         -f testthat.R \
 2>&1 \
 | tee out.log \
 | cat
 ```
 
+These tests can also be triggered on any pull request by leaving a "Comment" review with the following comment:
+
+> /gha run r-valgrind-tests
```

Comment:
`r-valgrind` for the shortness.

Reply:
changed in https://github.com/microsoft/LightGBM/pull/3443/commits/b03d8633ed919e919da5f4c526bfabd0c7e85d0e